README.md

```diff
@@ -76,7 +78,7 @@ Your output could be something like:
 It takes 5 entries for 3 happy days to occur
 ```
 
-**think**: What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case?
+**think**: What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case? Puts that there weren't 3 happy days (in the entries provided).  I handle that case after my loop?
```

Comment:
I'll chat to you all sometime about why this is weird to me as a Java programmer. Ruby lets you do it this way. But it's just a little strange to me. Ruby peeps prolly won't think it's weird.